src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DDMMYYYY = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String SEPARATOR = " ";
+    private static final String HEADER = "Report for period ";
+    private static final String DELIMITER = " - ";
+    private static final int ZERO = 0;
```

Comment:
It is necessary not just to name 1 and so on. It is necessary to specify what this index points to, for example DATE_INDEX